modules/EAPM/controller.php

```diff
@@ -125,7 +125,7 @@ protected function post_save()
             // It's OAuth, we have to handle this specially.
             // We need to create a new window to handle the OAuth, and redirect this window back to the edit view
             // So we will handle that in javascript.
-            $popup_warning_msg = string_format($GLOBALS['mod_strings']['LBL_ERR_POPUPS_DISABLED'], array($_SERVER['HTTP_HOST']));
+            $popup_warning_msg = strtr($GLOBALS['mod_strings']['LBL_ERR_POPUPS_DISABLED'], array( '{0}' => $_SERVER['HTTP_HOST']));
```

Comment:
Very "elegant" solution. Also there is the LangText::get() function which does something very similar for associative language text replacement at https://github.com/salesagility/SuiteCRM/blob/master/include/LangText.php#L329
I am not quite sure if it solves this issue with indexed parameters like "{0}". However, this PR solves the issue so I'll approve it now.

Reply:
Sorry, I'm having a hard time seeing it from quickly looking at that function.. I'm not sure what it does.  It would be great if it was better documented in the docBlock. I'll look and see how it's used elsewhere and if there is any testing showing how's it's used; however, are you signing off on my change as is or recommending I change it?